src/app/shared/components/template/components/pdf/pdf.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="pdf-viewer">
+  <ngx-extended-pdf-viewer
+    [src]="_row.value || src | plhAsset"
```

Comment:
nit(blocking)
I'm not sure of the order of operations here, probably better to resolve the source in the component ts and just have a single possible value